justfile

```diff
@@ -104,6 +104,17 @@ fix: devenv
     $BIN/black .
     $BIN/isort .
 
+
+# build the sqlrunner docker image
+build-sqlrunner:
```

Comment:
The convention we've used in other projects is `just docker-build`

That way, any project's docker images can be built with the same command

Reply:
Good to know, thank you. 1ee5a9c renames the just command.